plugins/lyrics-genius/menu.js

```diff
@@ -0,0 +1,17 @@
+const { setOptions } = require("../../config/plugins");
+const { toggleRomanized } = require("./back");
+
+module.exports = (win, options, refreshMenu) => {
+	return [
+		{
+			label: "Romanized Lyrics",
+			type: "checkbox",
+			checked: options.romanizedLyrics,
+			click: (item) => {
+				options.romanizedLyrics = item.checked;
+				setMenuOptions('lyrics-genius', options);
```

Comment:
Need to change that too

Reply:
am a bit sleepy so i will get to this in the morning